corehq/apps/users/models.py

```diff
@@ -2096,6 +2096,11 @@ def __repr__(self):
             self=self
         ))
 
+    @property
+    @memoized
```

Comment:
Yeah. I found it and mentioned it on #web-team a few weeks ago. Haven't gotten around to migrating our current usage over. I think the double decorator just comes from a time before django had implemented that (their implementation is also much much smaller/simpler than memoized since memoized also works for non-properties)